JS-Algorithms/challenges.js

```diff
@@ -70,7 +87,28 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  const modBase = Math.pow(10, lastDigits + 1);
+
+  const serie = Array.from({ length: number }, (_, idx) => idx + 1);
+
+  const poweredSerie = serie.map((num) => {    
```

Comment:
I know why you name this variable as num but maybe you can be more descriptive in your variables, don't be shy of using large naming, maybe `serieNumber` ?